packages/@lwc/engine-core/src/framework/hooks.ts

```diff
@@ -461,3 +755,185 @@ export function markAsDynamicChildren(children: VNodes) {
 function hasDynamicChildren(children: VNodes): boolean {
     return FromIteration.has(children);
 }
+
+function createKeyToOldIdx(
```

Comment:
@pmdartus was there any change to this block other than copying it from snabbdom.ts?

Reply:
This is a pure copy-paste. No change was made to this routine.